fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -823,13 +823,13 @@ private void appendInnerSelect(StringBuilder whereClauseSegment, QueryParameter
 
         whereClauseSegment.append(" WHERE ");
 
-        // CR1.RESOURCE_ID = CLR1.CURRENT_RESOURCE_ID AND CR1.IS_DELETED <> 'Y' AND
+        // CR1.RESOURCE_ID = CLR1.CURRENT_RESOURCE_ID AND CR1.IS_DELETED = 'N' AND
         // CP1.LOGICAL_RESOURCE_ID = CLR1.LOGICAL_RESOURCE_ID AND
         whereClauseSegment.append(chainedResourceTableAlias).append("RESOURCE_ID = ")
                 .append(chainedLogicalResourceTableAlias).append("CURRENT_RESOURCE_ID")
                 .append(AND)
                 .append(chainedResourceTableAlias)
-                .append("IS_DELETED").append(" <> 'Y'")
+                .append("IS_DELETED").append(" = 'N'")
```

Comment:
@michaelwschroeder FYI - the <> change, I think you might have already adopted this

Reply:
I've seen Mike has made this change when reviewing his PR.